iac-extensions/terraform/src/test/resources/checks/DisabledLoggingCheck/test_docdb_cluster.tf

```diff
@@ -1,11 +1,12 @@
 resource "aws_docdb_cluster" "sensitive_exports_value" {
   cluster_identifier = "DB Cluster With Logs"
   enabled_cloudwatch_logs_exports = ["profiler"] # Noncompliant {{Make sure that disabling logging is safe here.}}
-                                  # ^^^^^^^^^^^^
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 }
 
 # Noncompliant@+1 {{Omitting enabled_cloudwatch_logs_exports makes logs incomplete. Make sure it is safe here.}}
 resource "aws_docdb_cluster" "missing_exports_value" {
+  #      ^^^^^^^^^^^^^^^^^^^
```

Comment:
Minor: it doesn't look like this test case is in the scope of [SONARIAC-286](https://jira.sonarsource.com/browse/SONARIAC-286)